.github/workflows/helm-update-index.yaml

```
@@ -6,7 +6,7 @@ on:
       - '**.tgz'
 jobs:
   update-index:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

I'd suggest using a pinned version (22.04) to avoid unexpected issues once the latest tag points to v24.04
```suggestion
    runs-on: ubuntu-22.04
```

---

We'll revert if issue occurs. These are non critical jobs and we are already using latest tag for quite some time. Check helm-charts repo